charts/quickwit/values.yaml

```diff
@@ -38,49 +38,55 @@ securityContext:
 
 # Additional global env
 environment: {}
-  # KEY: VALUE
+# KEY: VALUE
```

Comment:
This comment shouldn't be changed, there is a tabs to keep below the environment hash.

Reply:
Fixed